packages/components/src/components/post-accordion-item/post-accordion-item.tsx

```diff
@@ -0,0 +1,90 @@
+import { Component, Element, h, Host, Listen, Method, Prop, State, Watch } from '@stencil/core';
+import { version } from '../../../package.json';
+import { checkEmptyOrOneOf } from '../../utils';
+import { HEADING_LEVELS, HeadingLevel } from './heading-levels';
+
+@Component({
+  tag: 'post-accordion-item',
+  styleUrl: 'post-accordion-item.scss',
+  shadow: true,
+})
+export class PostAccordionItem {
+  private collapsible: HTMLPostCollapsibleElement;
+
+  @Element() host: HTMLPostAccordionItemElement;
+
+  @State() id: string;
+  @State() isOpen: boolean;
+  @State() headingTag: string;
+
+  /**
+   * If `true`, the element is initially collapsed otherwise it is displayed.
+   */
+  @Prop() readonly collapsed?: boolean = false;
+
+  /**
+   * Defines the hierarchical level of the accordion item header within the headings structure.
+   */
+  @Prop() readonly headingLevel?: HeadingLevel = 2;
+
+  @Watch('headingLevel')
+  validateHeadingLevel(newValue = this.headingLevel) {
+    checkEmptyOrOneOf(
+      newValue,
+      HEADING_LEVELS,
+      'The `headingLevel` property of the `post-accordion-item` must be a number between 1 and 6.',
+    );
+  }
+
+  connectedCallback() {
+    this.validateHeadingLevel();
+  }
+
+  componentWillLoad() {
+    this.isOpen = !this.collapsed;
+  }
+
+  componentWillRender() {
+    this.id = this.host.id || `a${crypto.randomUUID()}`;
+    this.headingTag = `h${this.headingLevel ?? 2}`;
+  }
+
+  @Listen('collapseChange')
+  onCollapseChange(): void {
+    this.isOpen = !this.isOpen;
```

Comment:
Are you sure that a collapse change always inverts the state? Does the collapseChange event not send the actual current state as a parameter? Would be a good idea for stability to always assign what the event sends.

Also, how does this behave on nested accordions? Is the collapseChange event bubbling?

Reply:
For now, the event has no payload and is sent only when there is an actual change in the collapse. Should this be change?

 ![image](https://github.com/swisspost/design-system/assets/33580481/c79448c8-dc5e-45d2-a23e-d126e765d3f7)

Also, I stopped the event propagation so that nested accordions does not interfere.

Reply:
I would send the current state, this could be very helpful for other applications as well.

Reply:
I just updated the event.